frontend/packages/ceph-storage-plugin/src/components/modals/storage-class-dropdown.tsx

```diff
@@ -0,0 +1,48 @@
+import * as React from 'react';
+import * as _ from 'lodash';
+import { Firehose } from '@console/internal/components/utils';
+import { StorageClassDropdownInner } from '@console/internal/components/utils/storage-class-dropdown';
+import { K8sResourceKind } from '@console/internal/module/k8s';
+
+const cephStorageProvisioners = ['ceph.rook.io/block', 'cephfs.csi.ceph.com', 'rbd.csi.ceph.com'];
+
+export const OCSStorageClassDropdown: React.FC<OCSStorageClassDropdownProps> = (props) => (
+  <Firehose resources={[{ kind: 'StorageClass', prop: 'StorageClass', isList: true }]}>
+    <StorageClassDropdown {...props} />
+  </Firehose>
+);
+
+const StorageClassDropdown = (props) => {
+  const getFilteredStorageClassData = (storageClassData: K8sResourceKind[]): K8sResourceKind[] =>
+    storageClassData.filter(
```

Comment:
You're not using the return type? `filter` doesn't mutate the original array. It returns a new value.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/filter

Reply:
ack